docs/source/serialization.rst

```diff
@@ -430,3 +430,67 @@ Using the traced model for inference is as simple as using its ``__call__`` dund
 .. code-block:: python
 
     traced_model(tokens_tensor, segments_tensors)
+
+Using HuggingFace TorchScript model in AWS Inf1 using Neuron SDK
+-----------------------------------------------------------------------------------------------------------------------
+
+AWS introduced `Inf1 <https://aws.amazon.com/ec2/instance-types/inf1/>`_ for low cost, high performance machine learning model inference tasks in AWS cloud. 
+Inf1 is a custom-built high performance inference chip specializing in deep learning  
+inferencing workloads. The `AWS Neuron SDK <https://awsdocs-neuron.readthedocs-hosted.com/en/latest/#>`_ supports tracing models for high-performance deep 
+learning workloads in Inf1.` In Neuron SDK, there are these benefhts:
+
+
+1. Easy-to-use API with one line of code change to trace and optimize a TorchScript model for inference in the cloud.
+2. Out of the box performance optimizations for `improved cost-performance <https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-guide/benchmark/>`_ using the AWS Neuron.
+3. Support for HuggingFace (HF) models built with either `PyTorch <https://awsdocs-neuron.readthedocs-hosted.com/en/latest/src/examples/pytorch/bert_tutorial/tutorial_pretrained_bert.html>`_ or `TensorFlow <https://awsdocs-neuron.readthedocs-hosted.com/en/latest/src/examples/tensorflow/huggingface_bert/huggingface_bert.html>`_.
+
+
+Implications
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Models with transformer based architecture, i.e., 
+`BERT (Bidirectional Encoder Representations from Transformers) <https://huggingface.co/bert-base-uncased>`_ or its variants
+such as `distilBERT <https://huggingface.co/transformers/v4.0.1/model_doc/distilbert.html>`_ 
+and `roBERTa <https://huggingface.co/roberta-base>`_ will run best in Inf1 for non-generative tasks such as Extractive Question Answering, Sequence Classification, NER. 
```

Comment:
We should align the ref links to those models. I think adding the `docs` link for all 3 is probably better than the models weights on the hub or what do you think?

Reply:
Agreed and resolved. .DS_Store file removed, and links are all referring to the master docs links. Rebased and resubmitted to a new PR: https://github.com/huggingface/transformers/pull/14982